src/Uno.UI.RuntimeTests/MUX/Utilities/MUXTestPage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace MUXControlsTestApp.Utilities
+{
+    public partial class MUXTestPage : Page
+    {
+		public static List<T> FindVisualChildrenByType<T>(FrameworkElement parent) where T : class
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to use a generic collection designed for inheritance.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5156819)